### PR TITLE
Restore clang builds in travis.yml

### PR DESCRIPTION
**Pull request by richsalz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -32,21 +32,17 @@ env:
...
-#        - os: linux
-#          compiler: clang-3.6
-#          env: CONFIG_OPTS="no-asm enable-ubsan enable-rc5 enable-md2 -fno-sanitize=alignment"
         - os: linux
-          compiler: gcc-5
-          env: CONFIG_OPTS="no-shared no-asm enable-rc5 enable-md2"
+          compiler: clang-3.6
+          env: CONFIG_OPTS="no-asm enable-ubsan enable-rc5 enable-md2 -fno-sanitize=alignment"
+        - os: linux
+        compiler: clang-3.6
```
Comment (ekasper) [inline .travis.yml]: You have YAML errors - probably this line?

Comment (richsalz) [inline .travis.yml]: Dang, fixed.  Thanks.

**Comment by levitte** [comment]
Looks good to me.  As long as Travis agrees, I say ship.

**Comment by richsalz** [comment]
squashed and merged at 6f60a5a  closing.

**Pull request closed without merging**